ci/upload-launch.sh

```diff
@@ -6,17 +6,34 @@ cd "$(dirname "$0")/../"
 
 PREFIX="playground"
 
-ci/build-test.sh
-
 source ".shared-ci/scripts/profiling.sh"
 source ".shared-ci/scripts/pinned-tools.sh"
 
+if [[ -n "${BUILDKITE}" ]]; then
```

Comment:
We might want to add `isBuildKite` check if we intend to use this extensively

Reply:
Probably not worth it - all these checks can be removed once we are off TeamCity. Especially as it doesn't reduce the verbosity much at all